src/datasource/datasource.ts

```diff
@@ -68,6 +69,11 @@ export class TwinMakerDataSource extends DataSourceWithBackend<TwinMakerQuery, T
   }
 
   query(options: DataQueryRequest<TwinMakerQuery>): Observable<DataQueryResponse> {
+    for (const target of options.targets) {
+      if (target.queryType === TwinMakerQueryType.EntityHistory) {
```

Comment:
should this be moved to the else statement on line 96 so that it starts streaming after the AWS API finishes scrolling through the results?

```ts
          if (next.length) {
            return next;
          } else {
            // start streaming
          }
```

or is the idea to move scrolling to the backend for the queries that support streaming responses?